#!/bin/sh /usr/share/dpatch/dpatch-run
## 315-magic-add-mozilla.dpatch by Josh Triplett <josh@freedesktop.org>
##
## DP: New magic for Mozilla XUL fastload files,
## DP: commonly found in a Mozilla profile as XUL.mfasl and XPC.mfasl
## DP: (Closes: #452354).

@DPATCH@

diff -Naur file-4.21.orig/magic/Magdir/mozilla file-4.21/magic/Magdir/mozilla
--- file-4.21.orig/magic/Magdir/mozilla	1970-01-01 00:00:00.000000000 +0000
+++ file-4.21/magic/Magdir/mozilla	2007-12-23 17:13:40.000000000 +0000
@@ -0,0 +1,5 @@
+# Type:	Mozilla XUL fastload files (XUL.mfasl and XPC.mfasl)
+# URL:	http://www.mozilla.org/
+# From:	Josh Triplett <josh@freedesktop.org>
+
+0	string	XPCOM\nMozFASL\r\n\x1A		Mozilla XUL fastload data
